runtimes/common/Cargo.toml

```diff
@@ -40,6 +34,7 @@ pallet-authorship.workspace = true
 pallet-balances.workspace = true
 pallet-membership.workspace = true
 pallet-transaction-payment.workspace = true
+pallet-treasury.workspace = true
```

Comment:
This should be added also to the `std`, `runtime-benchmarks` and `try-runtime` features.

Reply:
Added: [a30a157](https://github.com/KILTprotocol/kilt-node/pull/527/commits/a30a15706cb14befb9c29e46e6d5ffc71e0f0249) and [1993e94](https://github.com/KILTprotocol/kilt-node/pull/527/commits/1993e94543f892d23cbc74de2698e6e88c673cd8)